src/genegraph/migration.clj

```diff
@@ -163,3 +169,28 @@
       (retrieve-migration env/genegraph-bucket archive-file env/data-vol)
       (decompress-database env/data-vol archive-path))))
 
+(defn load-stream-data
+  "Loads stream data from scratch into an existing database"
+  ([dest-path] (load-stream-data dest-path {}))
+  ([dest-path {from-scratch :from-scratch}]
+   (log/info :fn :load-stream-data :msg (str "Loading stream data into database at " dest-path))
+   (with-redefs [env/data-vol dest-path]
+     (stop #'event/stream-processing)
+     (populate-data-vol-if-needed)
+     ;(fs/mkdirs env/data-vol)
```

Comment:
Suggest `;` -=> `#_` or at least `;;`.
Why the `with-redefs` instead of passing the `dest-path`?
Why the blank line below?